koku/masu/processor/parquet/ocp_cloud_parquet_report_processor.py

```diff
@@ -189,7 +189,7 @@ def create_partitioned_ocp_on_cloud_parquet(self, data_frame, parquet_base_filen
         if self.provider_type == Provider.PROVIDER_AZURE and (
             date_field not in data_frame.columns or not data_frame[date_field].any()
         ):
-            date_field = "usagedatetime"
+            date_field = "date"
```

Comment:
I would think this entire if statement could be removed here right? Since the date_field we are checking will always be `date` now.